templates/docs/patterns/lists.md

```diff
@@ -123,6 +123,14 @@ If you wish to split the items in a list into two columns above `$breakpoint-sma
 View example of the patterns list split
 </a></div>
 
+## Nested Count
```

Comment:
This seems like a close variant of default list, so could you move it up on the page, right after the "default" section?

Reply:
Done :+1: